metrics/configs/flakes-experiment-config.yaml

```diff
@@ -75,9 +75,10 @@ query: |
                 select as struct
                   i
                 from t.metadata i
-                where (i.key = 'repos' or repos is not null) and
-                array_length(split(replace(ifnull(t.repos, i.value),', ', ''), ',')) = 2 /*serial pr jobs only (# of PR refs +1 == 2)*/
-              )
+                where (i.key = 'repos') and
+                array_length(split(replace(i.value,', ', ''), ',')) = 2 /*serial pr jobs only (# of PR refs +1 == 2)*/
+               )
+               or array_length(split(replace(t.repos,', ', ''), ',')) = 2
```

Comment:
I feel like this is a sign that pushing something `ifnull(t.repos, (select i.value from t.metadata i where i.key = 'repos')) as repos` into yet a deeper sub-select would make this part of the query more obvious.

Reply:
Will give it a go.